src/L1VotePool.sol

```diff
@@ -0,0 +1,104 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {IGovernor} from "openzeppelin/governance/Governor.sol";
+import {IWormhole} from "wormhole/interfaces/IWormhole.sol";
+
+import {WormholeReceiver} from "src/WormholeReceiver.sol";
+
+contract L1VotePool is WormholeReceiver {
+  /// @notice Must call castVote within this many blocks of the proposal deadline, so-as to allow
+  /// ample time for all depositors to express their vote preferences.
+  uint32 public constant CAST_VOTE_WINDOW = 1200;
```

Comment:
Highlighting this window, pulled from L2 flexible voting

Reply:
Removed https://github.com/ScopeLift/l2-flexible-voting/pull/9/commits/22726901a3dc4c204b69c912f675f0da9fe39fed

Reply:
https://github.com/ScopeLift/l2-flexible-voting/pull/9/commits/f37f8c41ece8f4cb723357f8cfe7a96d08031e7a